server/events/vcs/types/status.go

```diff
@@ -2,13 +2,17 @@ package types
 
 import "github.com/runatlantis/atlantis/server/events/models"
 
+type UpdateReqIdentifier struct {
+	Repo       models.Repo
+	Ref        string
+	StatusName string
+}
 type UpdateStatusRequest struct {
-	Repo models.Repo
+	UpdateReqIdentifier
```

Comment:
I initially thought of using `UpdateReqIdentifier` to match the `UpdateStatusReq` to a checkrun when we list all the checkruns for a repo using `ListCheckRunsForRef` method. However, since this method already takes in the commit-sha and the repo, we can just match the status name instead of {repo, sha, status-name}. So, I can get rid of this.